test/integration/integration.cc

```diff
@@ -249,6 +250,16 @@ BaseIntegrationTest::BaseIntegrationTest(Network::Address::IpVersion version,
   ON_CALL(factory_context_, api()).WillByDefault(ReturnRef(*api_));
 }
 
+BaseIntegrationTest::BaseIntegrationTest(Network::Address::IpVersion version,
+                                         const std::string& config)
+    : BaseIntegrationTest(
+          /*upstream_address_fn=*/
```

Comment:
nit: del

Reply:
My bad, thought I caught all of these. Done.